plugins/by-name/quarto-nvim/default.nix

```
@@ -0,0 +1,108 @@
+{
+  lib,
+  helpers,
+  ...
+}:
+with lib;
+with helpers.vim-plugin;
+mkVimPlugin {
+  name = "quarto-nvim";
```

```suggestion
  name = "quarto";
```
Also, move the source (and test) files to `quarto` instead of `quarto-nvim`.

---

Moved to quarto/ for the source and test